tiles/src/main/java/com/protomaps/basemap/layers/PhysicalLine.java

```diff
@@ -23,8 +23,16 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
         .setId(FeatureId.create(sf))
         .setAttr("waterway", sf.getString("waterway"))
         .setAttr("natural", sf.getString("natural"))
+        // Add less common attributes only at higher zooms
+        .setAttrWithMinzoom("bridge", sf.getString("bridge"), 12)
+        .setAttrWithMinzoom("tunnel", sf.getString("tunnel"), 12)
+        .setAttrWithMinzoom("layer", sf.getString("layer"), 12)
         .setZoomRange(12, 15);
 
+      if( sf.hasTag("intermittent", "yes" ) ) {
+        feat.setAttr("intermittent", 1);
```

Comment:
For way [316401077](https://www.openstreetmap.org/way/316401077) in **israel-and-palestine** GeoFabrik:

<img width="553" alt="image" src="https://github.com/protomaps/basemaps/assets/853051/511f3b66-051c-49d8-88b3-59e6b316c06b">

Reply:
@bdon up to you if you want this to be a boolean in the MVT or an int

Reply:
[Tilezen uses a boolean](https://tilezen.readthedocs.io/en/latest/layers/) so let's converge on that later